client/payment-details/summary/index.tsx

```diff
@@ -489,6 +489,12 @@ const PaymentDetailsSummary: React.FC< PaymentDetailsSummaryProps > = ( {
 									{ charge.payment_intent
 										? charge.payment_intent
 										: charge.id }
+									<br />
+									{ `${ __(
+										'Charge ID',
+										'woocommerce-payments'
+									) }: ` }
+									{ charge.id }
```

Comment:
It looks like charge id is a "legacy" value; the payment intent id field shows charge id if no pi.

This means that for certain transactions (old ones?), this new UI will show charge ID twice, mislabelled:

```
Payment ID: ci_123456
Charge ID: ci_123456
```

So I suggest: 

- Hide the whole element (label AND value) if the value is not available – for both pi and charge. i.e. add a conditional existence/non-empty check around the whole string. I.e. make each component robust to missing data.
- Fix the payment id logic so it only ever shows payment id - remove the fallback to `charge.id`.

Reply:
Done here add424773d8192089e3f31c9f0a22fe741aa477a.